src/plonk_util.rs

```diff
@@ -272,12 +272,12 @@ pub fn sigma_polynomials<F: Field>(
         .collect()
 }
 
-pub(crate) fn polynomial_degree<F: Field>(
+pub(crate) fn polynomial_degree_plus_1<F: Field>(
```

Comment:
Does this seem reasonable? The motivation was to avoid underflow in the case where the polynomial is all zeros, which came up with one of the constraint polynomials.

Reply:
Yes looks good.